static/js/hero.js

```diff
@@ -3,6 +3,43 @@
 /* global Hls */
 /* eslint-disable no-unused-vars */
 
+const backgroundVideoSel = "#background-video";
+const promoVideoSel = "#promo-video";
+
+function openVideoLightBox() {
+  const youtubeVideoSrc = $("#tv-light-box-yt-video").attr("data-href");
+  const hlsAboutVideoEl = $("video#tv-light-box-video");
```

Comment:
2 things:

1. When I change this to `$("video#tv-light-box-video").get(0)`, it still works. By that I mean that clicking "Watch Now" brings up the lightbox and plays the video I specified. Does that not work for you?
2. Even if that doesn't work for some reason, the `if` statement condition below is invalid. It will always evaluate to `true` even if there is no element that matches the `video#tv-light-box-video` selector. If my suggestion above doesn't work, then the condition below needs to be updated to `!youtubeVideoSrc && !hlsAboutVideoEl.length === 0` or `!youtubeVideoSrc && !hlsAboutVideoEl.get(0)`

Reply:
> 1- When I change this to $("video#tv-light-box-video").get(0), it still works. By that I mean that clicking "Watch Now" brings up the lightbox and plays the video I specified. Does that not work for you?

It does not work. First load successfully but clicking again over `Watch Now` button It is `undefined` for fancybox.

Reply:
Ah, you're right. Good catch. Let's change the conditional then